Mark9Mbugua/SendIT23
# Travis CI not running #161853712

#### What does this PR do?
It debugs Travis CI because the tests keep failing 

#### Description of Task to be completed?
-pip install coveralls
-pip freeze requirements,txt

#### How should this be manually tested?
clone this repo in a local repo.
make changes and commit to the relevant branches
observe if Travis CI tests have passed

#### What are the relevant pivotal tracker stories?

https://www.pivotaltracker.com/n/projects/2224004
#161853712
